sites/funmooc/src/backend/base/static/funmooc/js/xiti.js

```diff
@@ -0,0 +1,118 @@
+(function () {
+  /**
+   * 
+   * loadSmartTag
+   * Load ATInternet script then call the provided callback
+   * 
+   * @param number siteId 
+   * @param function callback 
+   */
+  function loadSmartTag(siteId, callback) {
+    var script = document.createElement('script');
+    script.type = "text/javascript";
+    script.async = true;
+    script.onload = callback;
+    script.onerror = function (error) { throw error };
+    script.src = "https://tag.aticdn.net/" + siteId + "/smarttag.js"
+    document.body.append(script)
+  }
+
+  function extractLevel2AndChapters(levels2) {
+    var chapters = location.pathname
+      .substr(4) // remove language slug
+      .split('/')
+      .filter(function (slug) { return slug.length > 0 });
+    var level2_slug = chapters.shift();
+
+    if (level2_slug.startsWith('cours') && chapters.length === 0) {
+      // We are on the search page.
+      level2_slug = 'search_engine';
```

Comment:
What's the point of replacing it if can recognize it as "cours"?

Reply:
It seems that a level2 has been created especially for search_engine page, so we should recognize as well.